pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -375,6 +401,215 @@ def _reraise_if_not_permitted(self, err: VerificationError, code: str, permits:
         self._logger.warn("Issue %s is permitted with justification: %s", err, justification)
         return justification
 
+    async def publish_client(self, working_dir, from_release_tag, release_name, arch, client_type):
```

Comment:
Let's be more clear about what `arch` means here. It seems to be a Brew arch instead of Go arch.